javaagent-spi/src/main/java/io/opentelemetry/javaagent/spi/exporter/SpanExporterFactory.java

```diff
@@ -32,4 +32,9 @@
    * @return The exporter names supported by this factory
    */
   Set<String> getNames();
+
+  /** Returns whether the batch span processor should not be used with this exporter. */
```

Comment:
Quick and dirty - I think better is to have a `otel_` flag to disable batch span processor since some people may want to do so.